src/js/components/RangeSelector/doc.js

```diff
@@ -72,7 +72,27 @@ export const doc = RangeSelector => {
 };
 
 export const themeDoc = {
-  'global.colors.controls': {
+  'global.colors.focus': {
+    description: 'TThe color when there is a focus',
+    type: 'string',
+    defaultValue: 'accent-1',
+  },
+  'global.edgeSize': {
+    description: 'The possible sizes for the margin, padding and gap',
+    type: 'object',
+    defaultValue: 'small',
+  },
+  'global.borderSize': {
+    description: 'The size of the border',
+    type: 'string',
+    defaultValue: 'undefined',
+  },
+  'global.colors.border': {
+    description: 'The color for the border',
+    type: 'string',
+    defaultValue: 'undefined',
```

Comment:
for `global.colors` we usually do have defaultValues.
in this case it should be 
```
defaultValue: {
      dark: 'rgba(255, 255, 255, 0.33)',
      light: 'rgba(0, 0, 0, 0.33)',
    },
```
you can find it on `base.js`